osu.Game/Overlays/SocialOverlay.cs

```diff
@@ -9,19 +9,23 @@
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Containers;
 using osu.Game.Graphics;
-using osu.Game.Graphics.Cursor;
 using osu.Game.Graphics.UserInterface;
 using osu.Game.Online.API;
 using osu.Game.Online.API.Requests;
 using osu.Game.Overlays.SearchableList;
 using osu.Game.Overlays.Social;
 using osu.Game.Users;
+using osu.Framework.Configuration;
+using osu.Framework.Threading;
+using System.Threading.Tasks;
 
 namespace osu.Game.Overlays
 {
-    public class SocialOverlay : SearchableListOverlay<SocialTab, SocialSortCriteria, SortDirection>, IOnlineComponent

```

Comment:
what was the reasoning for making this not an `IOnlineComponent`?

Reply:
Because I removed the `APIStateChanged` it was no longer necessary. I didn't want a repeat from the leaderboard with endless reloads if the API fails. That should only be temporarily though.